Lib/selectors.py

```diff
@@ -320,7 +320,7 @@ def select(self, timeout=None):
         timeout = None if timeout is None else max(timeout, 0)
         ready = []
         try:
-            r, w, _ = self._select(self._readers, self._writers, [], timeout)
+            r, w, _ = type(self)._select(self._readers, self._writers, [], timeout)
```

Comment:
Why is this needed?

Reply:
In CPython, built-in functions bind as methods to the module they're from, so `select.select` in CPython is a `method` bound to `select`. However, in RustPython, built-in functions don't do that (although that would be a good solution for accessing a function's module), so `self.select` binds `select.select` to `self` and then essentially calls `select.select(self, ...)`.

Reply:
Nevermind, I did a hack in the VM so this isn't necessary.